### PR TITLE
k8s: fix get_api_client usage in inventory plugin

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
changelogs/fragments/k8s_inventory.yml
plugins/inventory/k8s.py
plugins/module_utils/common.py

### CODE REVIEW

File: plugins/module_utils/common.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def configuration_digest(configuration):
     return digest
 
 
-def get_api_client(module=None):
+def get_api_client(module=None, **kwargs):
```
Comment (goneri) [inline plugins/module_utils/common.py:121]: Hi @Akasurde, can you explain why you need this? These parameters will be ignore.

**Review by goneri — CHANGES_REQUESTED** [review]

File: plugins/module_utils/common.py (hunk 1 above)
Comment (Akasurde) [reply plugins/module_utils/common.py:121]: Inventory plugin calls `get_api_client` with args, which fails the inventory plugin.

Comment (goneri) [reply plugins/module_utils/common.py:121]: Yes, but these arguments will be ignored anyway. Maybe we should just remove them?

Comment (Akasurde) [reply plugins/module_utils/common.py:121]: No they are used.

**Review by goneri — APPROVED** [review]
I would prefer to see the molecule fix in a different PR.

**Review by gravesm — APPROVED** [review]

**Pull request merged**